docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,163 @@
+---
+id: contributors-guide-to-open-sauced
+title: 'Contributors Guide to OpenSauced'
+sidebar_label: Contributors Guide to OpenSauced
+keywords:
+  - contributors
+  - guides
+---
+
+## Introduction to OpenSauced for Contributors
+
+OpenSauced is a platform dedicated to empowering you, the contributor, as you grow in your open source journey. We offer a GitHub-powered dashboard for tracking your contributions and gaining insights into your favorite open source projects. You can also customize your Insights and Lists Pages, connect with other contributors, and use our Highlights feature to display your contributions to the world.
```

Comment:
I think it's worth mentioning exploring repositories to find your next contribution as well. We should probably add in a section for how to use the Explore tab

Reply:
Good point @BekahHW, but wouldn't this be best as an explainer page? I mean, we don't want to be too detailed when mentioning the features here

Reply:
We want this to be a full guide for contributors. So you don't have to give an in-depth exploration of the feature, but it should be a part of the contributor journey bc that feature benefits contributors.

Idk if you watched the video I posted, but think about someone new coming to OpenSauced. This is like their welcome guide. Here are the tools, and here's how you can use them as a contributor. You can link to explainer pages, but explainer pages don't teach them how they should use it for their specific use case. So think about how you can dive deeper into the content.